main.js

```diff
@@ -16,3 +22,66 @@ closeMenu.addEventListener('click', close);
 hambuChilds.forEach((chld) => {
   chld.addEventListener('click', close);
 });
+
+seeproject1.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+seeproject2.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+seeproject3.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+seeproject4.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+// closewindow.addEventListener('click', () => {
+//   popupWindow.classList.remove('popup-active');
+// });
+
```

Comment:
- [ ] <b>REQUIRED:</b> Keep your code clean by making sure to remove all commented-out code.